console/src/components/Main/Main.js

```diff
@@ -614,7 +614,10 @@ class Main extends React.Component {
             </div>
           </div>
 
-          <div className={styles.main + ' container-fluid'}>
+          <div
+            id="right_container_wrapper"
```

Comment:
why is this needed? If needed, rename to main_wrapper. It is not wraapping right_container